.eslintrc

```diff
@@ -34,6 +34,7 @@
     "no-trailing-spaces": "error",
     "object-curly-spacing": ["error", "never"],
     "quotes": ["error", "double", {"allowTemplateLiterals": true, "avoidEscape": true}],
-    "semi": ["error", "never"]
+    "semi": ["error", "never"],
+    "no-constant-condition": ["error", {"checkLoops": false}]
```

Comment:
This sets off some alarm bells. Is there a less loop-prone implementation without loads of faff? The implementation makes sense, as long as that `queryResult.LastEvaluatedKey` _never_ comes back as something truthy. But if it does and we're below five results then we're in trouble.

Reply:
Turns out we didn't need this change anymore. Removed
efedddc24dd6d9a815081df296e328034d1c4120